provider/credentials_test.go

```diff
@@ -73,6 +73,7 @@ var _ = Describe("Credentials", func() {
 					"prometheus": {
 						"remote_read": [{
 							"url": "https://influxdb.aiven.io:2701/api/v1/prom/read?db=defaultdb",
+							"read_recent": true,
```

Comment:
I don't know if you've been able to test this change yet, but my current understanding is that `read_recent` is an optimisation, it shouldn't have any semantic effect.

https://github.com/prometheus/prometheus/issues/3599

Reply:
I've tested this on your Prometheus service

Before:
![image](https://user-images.githubusercontent.com/1482692/84924765-ec20bf80-b0c0-11ea-8bd4-5f1dcb6c4087.png)

After:
![image](https://user-images.githubusercontent.com/1482692/84924846-0490da00-b0c1-11ea-8817-c8908ee11421.png)

Reply:
Great, thanks!